### ISSUE TITLE
Better Formats Module

### ISSUE THREAD

**Issue by willykaram** [NYC-Camp/website#259]
I've installed and configured the Better Formats module twice now and somebody has disabled it twice. I have re-enabled the module based on the following use cases:
1. UX: We should be using this module in order to better control the format of text inputs that users have access to. There are many users who have no idea what the heck a "Text Format" is, so this helps to make the site more usable for them.
2. Avoiding Errors Clean-Up: On a few occasions text formats have been changed or users have not used the correct format and things end up being a mess on the front end of the site. We'd already had 3 instances of this already (1 raised on Twitter and 2 by email). We've also encountered this issue repeatedly in past years. 

IMO we should not be disabling modules that other team members have enabled without first checking to see if nobody objects. Additionally, we should ideally be more prudent than we're being about enabling some modules; however given the time frames, we simple need to be flexible in this regard (e.g. Fernando implemented the commerce functionality and we simply followed his lead in this regard). To an extent this is also a matter of trusting our teammates judgement to make the right call at this point in the game.

**Comment by esod** [comment]
I disabled the module when I was sorting out https://github.com/NYC-Camp/website/issues/254. I forgot to re-enable it. 

My hesitation with better_formats for Drupal 7 is from its project page. I quote directly.

 "and is <strong>under testing for a production stable release.</strong>" 

We simply don't use modules in this state of development. 

I meant no disrespect. Apologies.

**Comment by esod** [comment]
Please note that I didn't delete its configuration.

**Comment by likewhoa** [comment]
@willykaram @esod @everyone anytime you are enabling/disabling any modules on @live you should announce this on irc/skype so that anyone who may object can do so before hand. 

We really shouldn't be doing any of these things on live without first letting the whole team know and posting on irc/skype is a good place for this. We need more communication between us so that these things don't happen again.

@esod even as better_formats states that it needs more testing for a production release, there is already a beta out the only things I don't like about the project are
- seeks new maintainer
- beta release only

But, the things that are good are.
- better_formats was already in our repo so it was probably used last year
- last active commit was on March 2014
- only 11 open bugs out of 71 total
- 60K sites currently using it?
- 7.x usage is in par with 6.x so it's obviously stable enough for many

I think we can all learn from this that we need to communicate more when it comes to working on @live.

**Comment by willykaram** [comment]
@likewhoa we should leave this open until we fully address the point raised by Eric in #254 above. I should be able to help with that tonight as part of some of my other work.

**Comment by likewhoa** [comment]
@willykaram I don't think better_formats module was the cause of #254, the issue there was the use of wysiwyg which conflicted with ckeditor in the cod profile, since that module assumes you put ckeditor library in sites/all/libraries instead. IMO using wysiwyg is ok but since cod is tied in with ckeditor we might want to deal with that later.

**Comment by esod** [comment]
Exactly. It's two unrelated issues except when I was working on improving that layout (see the first screen shot in #254 if you need convincing) I disabled better_formats. I was unaware that better formats was not a stable release on a project I was working on earlier this year and I had to remove it. So it was instinctual that I would disable it. 

The usage numbers are convincing with better_formats, but that also might mean the bugs it may be introducing are substantial. In any case I just forgot to re-enable it when I got the layout under control. I absolutely didn't mean to seem like I was repeating what I did with the Chosen module that time. That really wasn't helpful.

What I need to follow up with is a pull request to remove the jquery_ui.module from our stack. jquery_ui.module was the real culprit since it prevented parsing the buttons for the CK Editor. Then once I got the buttons working I simply had to change the vertical tabs. 

Note that I consider that layout to be in passable shape (see the second screen shot in #254) so I'm not sure what you mean to do @willykaram when you say you "should be able to help with that tonight as part of some of my other work."
